src/_nebari/subcommands/init.py

```diff
@@ -565,6 +566,13 @@ def init(
             "-o",
             help="Output file path for the rendered config file.",
         ),
+        explicit: int = typer.Option(
```

Comment:
@Adam-D-Lewis is there any reason to have `int` over `bool` here? I know either of those will work but I'm wondering if there is any advantage.

Reply:
Making it an int allows you to specify `--explicit --explicit` or `-ee`  to have levels of explicitness.  We aren't using the levels yet, but may in the future.